labelbox/schema/webhook.py

```diff
@@ -61,20 +93,53 @@ def create(client, topics, url, secret, project):
 
         return Webhook(client, client.execute(query_str)["createWebhook"])
 
-    created_by = Relationship.ToOne("User", False, "created_by")
-    organization = Relationship.ToOne("Organization")
-    project = Relationship.ToOne("Project")
+    @staticmethod
+    def validate_topics(topics: List[WebhookTopic]):
+        if not isinstance(topics, list):
```

Comment:
will this fail is topics is a tuple? can you check if something is just iterable?

Reply:
A string is also an iterable. I'll change this to be `Iterable and not string`